documentation/book/ref-list-of-kafka-bridge-resources.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-bridge.adoc
+
+[id='ref-list-of-kafka-bridge-resources-{context}']
+= List of resources created as part of Kafka Bridge cluster
+
+The following resources will created by the Cluster Operator in the {ProductPlatformName} cluster:
```

Comment:
@laidan6000  missing 'be', but should really be "The following resources **are** created..."